bakerydemo/templates/blog/blog_index_page.html

```diff
@@ -2,48 +2,34 @@
 {% load wagtailcore_tags navigation_tags wagtailimages_tags %}
 
 {% block content %}
-    {% include "base/include/header-index.html" %}
+    {% if not tag %}
+        {% include "base/include/header-index.html" %}
+    {% endif %}
 
     <div class="container">
         {% if tag %}
             <div class="row">
                 <div class="col-md-12">
-                    <p>Viewing all blog posts by <span class="outline">{{ tag }}</span></p>
+                    <h1 class="index-header__title">Blog</h1>
+                </div>
+                <div class="col-md-12">
+                    <p class="index-header__introduction">Viewing all blog posts sorted by the tag <span class="blog-tags__tag">{{ tag }}</span>.</p>
                 </div>
             </div>
         {% endif %}
 
         {% if page.get_child_tags %}
-            <ul class="blog-tags tags list-inline">
+            <ul class="blog-tags">
                 {% for tag in page.get_child_tags %}
-                    <li><a href="{{ tag.url }}">{{ tag }}</a></li>
+                    <li><a class="blog-tags__pill" aria-label="Filter by tag name {{ tag }}" href="{{ tag.url }}">{{ tag }}</a></li>
                 {% endfor %}
             </ul>
         {% endif %}
 
-        <div class="row row-eq-height blog-list">
+        <div class="blog-list">
             {% if posts %}
                 {% for blog in posts %}
-                    <li class="col-xs-12 col-sm-6 col-md-3 blog-list-item">
-                        <a href="{% pageurl blog %}">
-                            <div class="image">
-                                {% image blog.image fill-850x450-c50 as image %}
-                                <img src="{{ image.url }}" width="{{ image.width }}" height="{{ image.height }}" alt="{{ image.alt }}" class="" />
-                            </div>
-                            <div class="text">
-                                <h2 class="blog-list-title">{{ blog.title }}</h2>
-                                <p>{{ blog.introduction|truncatewords:15 }}</p>
-                            </div>
-                            <div class="small footer">
-                                {% if blog.date_published %}
-                                    {{ blog.date_published }} by 
-                                {% endif %}
-                                {% for author in blog.authors %}
-                                    {{ author }}{% if not forloop.last %}, {% endif %}
-                                {% endfor %}
-                            </div>
-                        </a>
-                    </li>
+                    {% include "includes/card/blog-listing-card.html" %}
```

Comment:
Bit surprised here. Makes our cards less reusable and more error-prone to mandate them to always be li?